ironfish/src/network/peerNetwork.ts

```diff
@@ -78,6 +79,7 @@ export class PeerNetwork {
   readonly localPeer: LocalPeer
   readonly peerManager: PeerManager
   readonly onIsReadyChanged = new Event<[boolean]>()
+  readonly onTransactionSeen = new Event<[Transaction, Date]>()
```

Comment:
You can name these if you want, in the case that it's not clear from the event name what the parameter is.

```suggestion
  readonly onTransactionSeen = new Event<[Transaction, received: Date]>()
```